src/main/java/org/cyclonedx/maven/DefaultModelConverter.java

```diff
@@ -153,7 +153,9 @@ private String generatePackageUrl(String groupId, String artifactId, String vers
     }
 
     @Override
-    public Component convert(Artifact artifact, CycloneDxSchema.Version schemaVersion, boolean includeLicenseText) {
+    public Component convert(Artifact artifact, CycloneDxSchema.Version schemaVersion, boolean includeLicenseText, ExternalReference[] externalReferences) {
```

Comment:
as commented previously, no external reference parameter for dependencies artifacts

Reply:
Fixed in 20287587e8123fabf00afdacf6ab1cba581459be.